app/members/[slug]/page.tsx

```
@@ -8,30 +9,51 @@ import { getMemberBySlug, getMembers } from "@/lib/getMembers";
+    name,
+    ...(affiliation ? { jobTitle: affiliation } : {}),
+    url: `https://latina.dev/members/${slug}`,
+    image: `https://latina.dev/img/members/${slug}.jpg`,
+    ...(sameAs.length > 0 ? { sameAs } : {}),
+  };
+
   return (
     <div className="w-full pt-20">
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(personJsonLd) }}
```

Biome (2.4.6)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)